recipe/meta.yaml

```diff
@@ -10,15 +10,23 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  # See conda-forge/glib-feedstock#40 for documentation regarding the RPATH patch
+  patches:
+    - 0001-Only-fix-RPATH-if-install_rpath-is-not-empty.patch
 
 build:
   number: 0
-  skip: True # [py<37]
-  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  script: {{ PYTHON }} -m pip install . -vvv --no-deps
+  skip: True  # [py<36]
```

Comment:
It supports python >=3.7 https://github.com/mesonbuild/meson/blob/f341097c7e7914c250133bd58feb4985d1fbdcd6/setup.py#L19

Reply:
hmm, I checked what conda-forge sees as compatible.  But sure, we can go without 3.6